core/modules/path/path.module

```diff
@@ -593,10 +593,9 @@ function path_config_info() {
  * Get all path information from modules implementing hook_path_info().
  */
 function path_get_info() {
-  $all_info = backdrop_static(__FUNCTION__);
+  $all_info = backdrop_static(__FUNCTION__, array());
```

Comment:
@jayelless as @bugfolder skipped that function in his ampersand PR, you'd have to add it here.

`&backdrop_static(`